### PR TITLE
Alex commit 

### PR DESCRIPTION
**Pull request by Arditti93**

### CODE REVIEW

File: alex.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const hal900 = () => {
```
Comment (nmunro) [inline alex.js:1]: Isn't it hal 9000?

Comment (SilviaPetruta) [inline alex.js:1]: Nice 👍

File: alex.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const hal9000 = () => {
```
Comment (nmunro) [inline alex.js:1]: Yup that's fixed it! Thanks!

Comment (SilviaPetruta) [inline alex.js:1]: Ohh.. I see

**Pull request merged**